packages/turf-line-overlap/test/out/issue-#901.geojson

```diff
@@ -59,34 +36,11 @@
       "geometry": {
         "type": "LineString",
         "coordinates": [
-          [-113.33832502043951, 53.52244398828247],
-          [-113.3384152645109, 53.52244409344282],
           [-113.33847575084239, 53.52244416392682],
-          [-113.3384152645109, 53.52244409344282]
-        ]
-      },
-      "bbox": [
-        -113.3384152645109,
-        53.52244398828247,
-        -113.33832502043951,
-        53.52244409344282
-      ],
-      "id": 20
-    },
-    {
-      "type": "Feature",
-      "properties": {
-        "stroke": "#0F0",
-        "fill": "#0F0",
-        "stroke-width": 25
-      },
-      "geometry": {
-        "type": "LineString",
-        "coordinates": [
+          [-113.3384152645109, 53.52244409344282],
           [-113.33832502043951, 53.52244398828247],
           [-113.3384152645109, 53.52244409344282],
```

Comment:
We're getting some duplicate coords here, eg 0 & 2 are the same, 1 & 3 are the same...

Reply:
I agree that this is still an issue, but in the current master branch the output of this testcase also contains duplicate coords and in addition to that the current master duplicates those linestrings. So I do not consider this to be a regression but a slight improvement. As I built this patch mainly to fix bugs where output was missing entirely, I do not think completely solving the existing duplicate coordinate issues in the output belongs here, but should rather go in a new patch. I'm willing to create that as well in the near future.